elasticsearch/run.sh

```diff
@@ -28,4 +39,24 @@ else
 	exit 1
 fi
 
-exec /usr/share/elasticsearch/bin/elasticsearch
+ln -s /usr/share/elasticsearch/config/${CLUSTER_NAME}.truststore /usr/share/elasticsearch/config/${CLUSTER_NAME}.truststore.jks
+cp /etc/elasticsearch/secret/admin-jks /usr/share/elasticsearch/config/admin.jks
+
+TIMES=60
+function waitForES() {
+  for (( i=1; i<=$TIMES; i++ )); do
+    # test for ES to be up first
+		# don't provide a client cert since ES doesn't validate us correctly
```

Comment:
@lukas-vlcek no, so this is specific to the work for es 2.3 due to how SG now initializes itself. It basically blocks traffic to ES unless it is configured which is what this loop is ensuring (we set up the openshift-elasticsearch-plugin to check if its initialized when a rest request comes in)